.github/dependabot.yml

```
@@ -0,0 +1,11 @@
+version: 2
+updates:
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    ignore:
+      # Offical actions have moving tags like v1
```

```suggestion
      # Official actions have moving tags like v1
```

Pretty sure this is my typo :)

---

These things are coming directly from scikit-hep/cookie, which is why I had overlooked recursive checkouts.

I'll apply all of these as a single commit on my side. Thanks!